Tests/DangerTests/GitDiffTests.swift

```diff
@@ -30,20 +38,31 @@ final class GitDiffTests: XCTestCase {
                                    FileDiff.Line(text: "gem \"xcpretty-json-formatter\"", changeType: .removed),
                                ]),
                            ]),
-                           FileDiff(parsedHeader: .init(filePath: "Sources/DangerSwiftCoverage/Models/Report.swift", change: .modified), hunks: [
+                           FileDiff(parsedHeader: .init(
+                               filePath: "Sources/DangerSwiftCoverage/Models/Report.swift",
+                               change: .modified
+                           ),
+                                    hunks: [
                                .init(oldLineStart: 20, oldLineSpan: 7, newLineStart: 20, newLineSpan: 8, lines: [
                                    FileDiff.Line(text: "extension ReportSection {", changeType: .unchanged),
-                                   FileDiff.Line(text: "    init(fromSPM spm: SPMCoverage, fileManager: FileManager) {", changeType: .unchanged),
+                                   FileDiff.Line(text: "    init(fromSPM spm: SPMCoverage, fileManager: FileManager) {",
+                                                 changeType: .unchanged),
                                    FileDiff.Line(text: "        titleText = nil", changeType: .unchanged),
-                                   FileDiff.Line(text: "        items = spm.data.flatMap { $0.files.map { ReportFile(fileName: $0.filename.deletingPrefix(fileManager.currentDirectoryPath + \"/\"), coverage: $0.summary.percent) } }", changeType: .removed),
-                                   FileDiff.Line(text: "        items = spm.data.flatMap { $0.files.map { ReportFile(fileName: $0.filename.deletingPrefix(fileManager.currentDirectoryPath + \"/\"), coverage: $0.summary.percent)", changeType: .added),
+                                   FileDiff.Line(text: "        items = spm.data.flatMap { $0.files.map { ReportFile(fileName: $0.filename.deletingPrefix(fileManager.currentDirectoryPath + \"/\"), coverage: $0.summary.percent) } }",
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitDiffTests.swift;
  Line: 51;
-->

- :warning: Line should be 140 characters or less: currently 232 characters (`line_length`)